src/store/getters/state-getters.ts

```diff
@@ -680,3 +681,8 @@ export const getCompletingParty = (state: StateIF): CompletingPartyIF => {
 export const getDissolutionDate = (state: StateIF): string => {
   return getDissolution(state).dissolutionDate
 }
+
+// parties List
```

Comment:
If you use JSDoc format (like the others) then you get context-sensitive help for this getter.